### PR TITLE
Posts are visible starting with newest

### PR DESCRIPTION
**Pull request by anaalta**
- added feature test checking the order of posting
- added orderly gem
- added post controller test to check the order of posts in the Post.all
- in posts/index, posts displayed in reverse
- added web helper for creating a post

### CODE REVIEW

File: spec/controllers/posts_controller_spec.rb
Comment (dbugsy) [inline spec/controllers/posts_controller_spec.rb]: There is a gem called 'Factory Girl' that can make this a bit less brittle

**Comment by dbugsy** [comment]
Consider adding Factory Girl to manage your model fixtures. Otherwise :shipit: :shipit:

**Review by FiddlersCode — CHANGES_REQUESTED** [review]
Looks great!  A few small things:
posts_controller_spec: 
- remove commented code in lines 20 and 28
- line 31 looks to be missing and `end` or should it be part of a previous describe block?

spec/webhelpers:
We have updated the front-end so you once the front-end branch is merged, you'll need `click_button "New Post"` (note uppercase "p"

**Review by milesillsley — COMMENTED** [review]
I looked over this with Paula, can approve after changes.

**Comment by anaalta** [comment]
@pmuldoon86, @milesillsley  thanks for the review! deleted the commented code, put an it under a describe block & clicking on the right New Post button now :)

**Comment by chrisjmit** [comment]
@anaalta @BenNoonan1991 could you resolve the conflicts please so we can pull the repo for testing? thanks

File: spec/controllers/posts_controller_spec.rb
Comment (dbugsy) [inline spec/controllers/posts_controller_spec.rb]: This should be in spec/factories/posts.rb as per: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#defining-factories

**Review by dbugsy — APPROVED** [review]
One small change, then I'm happy for you to merge :shipit: :shipit:

**Review by danielemanuel — APPROVED** [review]
:shipit:

**Pull request merged**
